src/components/Checkbox/PCheckbox.vue

```diff
@@ -81,6 +83,11 @@
   w-auto
 }
 
+.p-checkbox--small .p-checkbox__input { @apply
+  h-3
+  w-3
+}
```

Comment:
The checkbox itself gets smaller? They're pretty small as is aren't they? 

Also `p-checkbox__input--small`